css/home.css

```diff
@@ -102,53 +96,407 @@ Community Partnerships
+.wwd-card {
+	width: calc(33% - var(--spacing-med));
+}
+
+.wwd-card-img img {
+	border-radius: var(--border-radius) var(--border-radius) 0 0;
+	object-position: top;
+	object-fit: cover;
+	height: 200px;
+}
+
+.wwd-card-text {
+	border-radius: 0 0 var(--border-radius) var(--border-radius);
+	padding: var(--spacing-med) var(--spacing-sm);
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+	min-height: 330px;
+}
+
+/* individual card styling */
+.wwd-digital {
+	background-color: var(--color-teal);
+}
+.wwd-community {
+	background-color: var(--color-copper);
+}
+.wwd-programs {
+	background-color: var(--color-blue);
+}
+
+/* 
+===================================
+Our Values
+===================================
+*/
+
+/* text adjustments */
+
+.value-card h4 {
+	font-size: var(--font-size-title-med);
+}
+
+/* section styles */
+.our-values {
+	color: var(--text-color-light);
+	background: url("../assets/images/pages/home/bg-section-our-values.jpg")
+		top/cover;
+}
+
+/* title and card layout */
+.our-values .wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-med);
+}
+/* title layout */
+.values-title-container {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+/* values list layout */
+.values-list {
+	display: flex;
+	justify-content: space-between;
+	background-color: var(--color-blue);
+
+	border-radius: var(--border-radius);
+	padding: var(--spacing-lg);
+	gap: var(--spacing-lg);
+}
+/* card style */
+.value-card {
+	width: calc(33% - var(--spacing-lg));
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Upcoming Events and Programs
+===================================
+*/
+
+/* text adjustments */
+.events,
+.event a {
+	color: var(--text-color-light);
+}
+
+/* section background */
+.events {
+	background: url("../assets/images/pages/home/bg-overlay-vector-02.svg")
+			center/cover,
+		linear-gradient(to top, var(--color-grape), var(--color-blue));
+}
+/* event list layout */
+.event-list {
+	margin-top: var(--spacing-med);
+
+	display: flex;
+	justify-content: space-between;
+	gap: var(--spacing-sm);
+}
+
+/* Event Card Style */
+.event {
+	width: calc(33% - var(--spacing-sm));
+	max-width: 320px;
+}
+
+.event img {
+	border-radius: var(--border-radius);
+	min-height: 430px;
+	object-fit: cover;
+}
+
+.event-text {
+	margin-top: var(--spacing-sm);
+	display: flex;
+	flex-direction: column;
+	gap: 20px;
 }
 
 /* 
 ===================================
 Media Queries (overriding styles for different screen sizes)
+===================================
+*/
 
-Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
-Start by writing styles for the mobile version of the design on the lines above.
-Anything that needs to change for larger screens should be added to the media queries below.
+/* Tablet and smaller 1024px*/
+@media screen and (max-width: 1024px) {
```

Comment:
More static values for media query work - will do some research why these vars are not working well!